soroban-env-host/src/host.rs

```diff
@@ -1839,6 +1839,13 @@ impl VmCallerEnv for Host {
         min: U32Val,
     ) -> Result<Void, HostError> {
         let key = self.contract_data_key_from_rawval(k, t.try_into()?)?;
+
+        // We don't load this key, but we want to make sure that it's in the footprint.
```

Comment:
Is it correct that core reads the key? If so, can we say that?
```suggestion
        // The host doesn't load the key, but we want to make sure that
        // it's in the footprint, because the system embedding the host
        // (e.g. stellar-core) will to perform the bump.
```